src/Elcodi/Component/Media/Tests/UnitTest/Adapter/Resizer/Abstracts/AbstractResizeAdapterTest.php

```diff
@@ -58,6 +58,17 @@ public function testResize(
         imagejpeg($image, $imagePath, 100);
         $imageData = file_get_contents($imagePath);
 
+        if (!file_exists('/usr/bin/convert') || !file_exists('/usr/share/color/icc/colord/sRGB.icc')) {
```

Comment:
But this is the abstract adapter!
We must add these lines in the  `ImageMagickResizeAdapterTest` file, before returning the array in method `getAdapterInstance`.